### PR TITLE
添加VMess自定义任意门

### PR DESCRIPTION
**Pull request by Saxon-Sun**
按自己的配置写了个Vmess任意门
测试了没问题
大佬看看哪里需要修改一下

### CODE REVIEW

**Comment by mack-a** [comment]
提交到dev分之，合并一下commit

**Comment by mack-a** [comment]
outboundTag 写 VMess-out

File: install.sh [hunk 1]
```diff
@@ -3887,7 +3887,78 @@ setDokodemoDoorUnblockStreamingMediaOutbounds() {
...
 		setDokodemoDoorUnblockStreamingMediaOutbounds
 	fi
+	
+	echoContent yellow "1.使用默认配置"
+	echoContent yellow "2.使用VMESS"
+	read -r -p "请选择:" selectType
+
+	case ${selectType} in
+	1)
+		setDokodemoDoorUnblockStreamingMediaOutboundsdefault
```
Comment (mack-a) [inline install.sh:3897]: setDokodemoDoorUnblockStreamingMediaOutboundsDefault 改成这个

File: install.sh [hunk 2]
```diff
@@ -3887,7 +3887,78 @@ setDokodemoDoorUnblockStreamingMediaOutbounds() {
...
+		;;
+	2)
+		setDokodemoDoorUnblockStreamingMediaInboundsVMESS
```
Comment (mack-a) [inline install.sh:3900]: setDokodemoDoorUnblockStreamingMediaInboundsVMess

File: install.sh [hunk 3]
```diff
@@ -3887,7 +3887,78 @@ setDokodemoDoorUnblockStreamingMediaOutbounds() {
...
+	fi
+	
+	unInstallOutbounds Vmess
+	
+	outbounds=$(jq -r ".outbounds += [{\"tag\":\"Vmess\",\"protocol\":\"vmess\",\"streamSettings\":{\"network\":\"ws\",\"security\":\"tls\",\"tlsSettings\":{\"allowInsecure\":false},\"wsSettings\":{\"path\":\"ws\"}},\"mux\":{\"enabled\":true,\"concurrency\":8},\"settings\":{\"vnext\":[{\"address\":\"${setIP}\",\"port\":${VMessPort},\"users\":[{\"id\":\"${VMessID}\",\"security\":\"auto\",\"alterId\":0}]}]}}]" ${configPath}10_ipv4_outbounds.json)
+	echo "${outbounds}" | jq . >${configPath}10_ipv4_outbounds.json
+	
+	if [[ -f "${configPath}09_routing.json" ]]; then
+	
+	unInstallRouting Vmess outboundTag
```
Comment (mack-a) [inline install.sh:3930]: unInstallRouting VMess-out outboundTag

File: install.sh [hunk 4]
```diff
@@ -3887,7 +3887,78 @@ setDokodemoDoorUnblockStreamingMediaOutbounds() {
...
+  "routing": {
+    "rules": [
+      {
+        "type": "field",
+        "domain": [
+          "ip.sb",
+          "geosite:${domainList//,/\",\"geosite:}"
+        ],
+        "outboundTag": "Vmess"
+      }
```
Comment (mack-a) [inline install.sh:3949]: "outboundTag": "VMess-out"

File: install.sh [hunk 5]
```diff
@@ -3887,7 +3887,78 @@ setDokodemoDoorUnblockStreamingMediaOutbounds() {
...
+}
+EOF
+		fi
+		reloadCore
+		echoContent green " ---> 添加出站解锁成功"
+		exit 0
+fi
+}
 
+setDokodemoDoorUnblockStreamingMediaOutboundsdefault() {
```
Comment (mack-a) [inline install.sh:3961]: setDokodemoDoorUnblockStreamingMediaOutboundsDefault

File: install.sh [hunk 6]
```diff
@@ -4084,12 +4154,14 @@ removeDokodemoDoorUnblockStreamingMedia() {
 
 	unInstallOutbounds streamingMedia-80
 	unInstallOutbounds streamingMedia-443
+	unInstallOutbounds Vmess
```
Comment (mack-a) [inline install.sh:4157]: unInstallOutbounds VMess-out

File: install.sh [hunk 7]
```diff
@@ -4084,12 +4154,14 @@ removeDokodemoDoorUnblockStreamingMedia() {
...
 
 	unInstallRouting streamingMedia-80 inboundTag
 	unInstallRouting streamingMedia-443 inboundTag
 
 	unInstallRouting streamingMedia-80 outboundTag
 	unInstallRouting streamingMedia-443 outboundTag
+	unInstallRouting Vmess outboundTag
```
Comment (mack-a) [inline install.sh:4164]: unInstallRouting VMess-out outboundTag

**Comment by Saxon-Sun** [comment]
妥大佬 我明早改完提交到dev分支

**Pull request closed without merging**